kcidb/cache/__init__.py

```diff
@@ -52,11 +52,13 @@ def store(self, url):
 
         # Cache every 4096th URL only for the trial period
         if not object_name.endswith("000"):
+            LOGGER.debug("Skipping URL '%s', hash not matched.", url)
```

Comment:
Please remove this one, as this would generate *waaaay* too many log messages (as I mentioned on our call).